benchmarks/routines/gemm.py

```
@@ -836,6 +842,235 @@ def run_backend(backend, input_fp8, mat2_fp8, input_inv_s, mat2_inv_s):
+            backends_to_remove.append(backend)
+            continue
+
+        try:
+            flashinfer.mm_fp8(
+                input_fp8,
+                mat2_prepared,
+                alpha,
+                out_dtype=res_dtype,
+                backend=backend,
+            )
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)